docs: documentation for Mend-hosted Renovate plans (#33451)
renovatebot/renovate: docs/usage/mend-hosted/renovate-plans.md

# Renovate Plans on Mend-Hosted Apps

Mend provides cloud hosting services for running Renovate on repositories hosted on the following cloud platforms:

- GitHub

> - Bitbucket

Reviewer: ```suggestion
- Bitbucket Cloud
```

- Azure DevOps

Mend Cloud will regularly schedule Renovate jobs against all installed repositories.

> It also listens to changes on the repos and triggers a Renovate job when changes occur in the repo,

Reviewer: ```suggestion
It also listens to webhooks and enqueues a Renovate job when relevant changes occur in a repo,
```

> or when actions are triggered from the Renovate PRs or Dashboard issue.

Author: Done
